test/utils/driver.js

```diff
@@ -72,6 +72,40 @@ const Driver = {
     this.app = this.port = this.fixturesURL = this._stopServer = this._bwHandle = null;
   },
 
+  /**
+   * Fetches the redux state of the first browser window and returns
+   * it as a plain JS object (de-immutable-ified).
+   *
+   * @return Promise<Object>
+   */
+  getReduxState: async function() {
+    await this.setTargetToBrowserWindow();
+
+    return await this.client.execute(() =>
+      JSON.stringify(require('electron').ipcRenderer.store.getState()))
```

Comment:
It's worth communicating that the Redux state does not transmit via JSON cleanly.  We will want `transit` and `transit-immutable` to rehydrate meaningful record types.